requirements.txt

```diff
@@ -1,9 +1,9 @@
 build==0.4.0
 certifi==2021.5.30
 chardet==4.0.0
-grpcio>=1.37.1
-grpcio-testing>=1.37.1
-grpcio-tools>=1.37.1,<=1.43.0
+grpcio==1.47.0
+grpcio-testing==1.47.0
+grpcio-tools==1.47.0
```

Comment:
Please also change these versions in `setup.py`.

Reply:
By the way, any suggestions why did we need this?

Reply:
> By the way, any suggestions why did we need this?

https://github.com/grpc/grpc/issues/24069 https://github.com/grpc/grpc/pull/29050